src/reducers/permissions/actions.js

```diff
@@ -12,12 +12,12 @@ export const UPDATE_PERMISSIONS = PREFIX + 'UPDATE_PERMISSIONS'
 export const requestPermission = (permission: Permission) => (dispatch: Dispatch, getState: GetState) => {
   RNPermissions.check(permission).then(permissionStatus => {
     request(permission)
-      .then(status => dispatch(updatePermissions({ [permission]: status })))
+      .then(status => dispatch(updatePermissions({ [permission]: status }))) // eslint-disable-line no-undef
       .catch(console.log)
   })
 }
 // $flowFixMe
-export const updatePermissions = (permissions: { [Permission]: any }) => ({
+export const updatePermissions = (permissions: Object) => ({
```

Comment:
This is kind of lame as you used to have a more specific PermissionStatus object type. I realize it's not as bad as `any` but using `Object` is a bit of a cop out. I'll let it slide but I'll be asking later why you couldn't resolve this.